### DISCUSSION TITLE
Auto delete image file on single image deletion (not on model deletion)

### DISCUSSION THREAD

**Discussion by dev-dev-15** [filamentphp/filament#8174] [Help]
To delete the images of a deleted model we can use an Observer, but what about when deleting single images from here:
![image](https://github.com/filamentphp/filament/assets/14108109/c57a7ec1-60fd-4e90-839f-86ba31370535)

**Comment by pboivin** [comment]
I think this can also be done with observers. You could use a mix of `updating` and `updated` to determine which images need to be deleted:
https://laravel.com/docs/10.x/eloquent#events

**Reply by dev-dev-15** [reply]
I'll try it, thanks
